testing/regress/ecl/genjoin.ecl

```diff
@@ -25,8 +25,10 @@ jrec := RECORD,MAXLENGTH(100)
     UNSIGNED1 i;
     STRING3 lstr;
     STRING3 rstr;
-    UNSIGNED1 c;
```

Comment:
See line 18:

 // IMPORTANT: this test is generated by the perl script genjointest.pl, so do not edit it by hand